components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/impl/MLProjectHandler.java

```diff
@@ -71,6 +71,14 @@ public MLProject getProject(int tenantId, String userName, String projectName) t
             throw new MLProjectHandlerException(e.getMessage(), e);
         }
     }
+
+    public MLProject getProject(int tenantId, String userName, long projectId) throws MLProjectHandlerException {
+        try {
+            return databaseService.getProject(tenantId, userName, projectId);
+        } catch (DatabaseHandlerException e) {
```

Comment:
@nuwand this class is just a delegator and the DatabaseHandlerException is wrapped to MLProjectHandlerException. MLProjectHandlerException will be logged at the upstream and DatabaseHandlerException is logged at the downstream. Do we still need to log it here too? Isn't it just make it a long stack trace?